### PR TITLE
Sensorhub

### PR DESCRIPTION
**Pull request by BabyCakes13**
Refactored and implemented some stuff:
- EnvironmentalSensor class now includes BME_680 support, data is collected as a dictionarry and other perks;
- Added a SensorHub which will be the handler of all the types of sensors (yet to be implemented: motion, light);
- Fixed typo in requirements and disabled it because of too much time wasted at testing it.

### CODE REVIEW

File: DataSender.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import paho.mqtt.client as mqtt
 import socket
 
+
```
[the remark this answers is not in the document]
Comment (BabyCakes13) [reply DataSender.py]: It's a PEP8 requirement for python to have two brake line between the last import and the class.

File: EnvironmentalSensor.py [hunk 2]
```diff
@@ -3,44 +3,74 @@
...
 import adafruit_bme280
+import adafruit_bme680
 
-addresses = [0x77, 0x76]
 
-sensors = []
+class EnvironmentalSensor:
+    I2C = busio.I2C(board.SCL, board.SDA)
+    ADDRESSES = [0x77, 0x76]
+    SEA_LEVEL_PRESSURE = 1013.25
```
[the remark this answers is not in the document]
Comment (BabyCakes13) [reply EnvironmentalSensor.py]: I'll take care of it.

File: EnvironmentalSensor.py [hunk 3]
```diff
@@ -3,44 +3,74 @@
...
+    def collect_data(self) -> dict:
+        """
+        Collect environmental data from the sensor and update self.data_packet
+        :return: None
+        """
+        data_packet = {}
 
-# change this to match the location's pressure (hPa) at sea level
-sensor.sea_level_pressure = 1013.25
+        if self.sensor is not None:
```
[the remark this answers is not in the document]
Comment (BabyCakes13) [reply EnvironmentalSensor.py]: Yep.

File: requirements.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
+board
```
Comment (alex-sever-h) [inline requirements.txt]: why is board listed as a requirement ? 
it is a pre-installed package on raspberry, as far as I know

File: run.sh [hunk 5]
```diff
@@ -6,13 +6,13 @@ rsync -aAv --progress --delete `pwd`/ pi@raspberry-sensor-dev:~/HomeSensorHub_$U
 
 case "$1" in
     "env")
-        ssh -t pi@raspberry-sensor-dev python3 /home/pi/HomeSensorHub_$USER/EnvironmentalSensor.py
+        ssh -t pi@raspberry-sensor-dev python3 /home/pi/HomeSensorHub_$USER/main.py
```
Comment (alex-sever-h) [inline run.sh]: env should remain to launch only env, not main.
please make another case for main.py

File: requirements.txt (hunk 4 above)
Comment (BabyCakes13) [reply requirements.txt]: Didn't know. I was writing the code on my local machine where python was giving me the warning that i don't have board installed. Gonna remove it.

**Pull request merged**